db/change_cache_test.go

```diff
@@ -324,7 +324,7 @@ func TestLateSequenceErrorRecovery(t *testing.T) {
 	// Modify the cache's late logs to remove the changes feed's lateFeedHandler sequence from the
 	// cache's lateLogs.  This will trigger an error on the next feed iteration, which should trigger
 	// rollback to resend all changes since low sequence (1)
-	c, err := dbCollection.changeCache.getChannelCache().getSingleChannelCache(channels.NewID("ABC", collectionID))
+	c, err := dbCollection.changeCache().getChannelCache().getSingleChannelCache(channels.NewID("ABC", collectionID))
```

Comment:
This could be simplified to db.changeCache as below.

Reply:
fixed.